schema/v1.1.0/dataset_config_models_extended.py

```diff
@@ -0,0 +1,321 @@
+from __future__ import annotations
```

Comment:
A non-auto generated file. This is where the extended validation (validation that linkml can't do) is written, based closely off the classes of `dataset_config_models.py`